OrbitCore/ContextSwitch.h

```diff
@@ -3,6 +3,8 @@
 //-----------------------------------
 #pragma once
 
+#include "Serialization.h"
+
 //-----------------------------------------------------------------------------
```

Comment:
#pragma pack(push, 1)/#pragma pack(pop) could minimize the risk of having different memory layout on different platforms/compilers.

Reply:
That might reduce the risk, but for sure it will not delete it. See [comment](https://github.com/pierricgimmig/orbitprofiler/pull/90#discussion_r357026518). Shouldn't we use the binary serialization instead?

Reply:
I think that for a simple POD type like this one, we could simply send raw data, especially since there will be thousands of context switches per second.